core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -37,6 +36,9 @@ const (
 	defaultAllowListExpiration = 20 * time.Minute
 	// allowListCleanupInterval decides when the expired items in allowList cache will be deleted.
 	allowListCleanupInterval = 5 * time.Minute
+	// TODO decide on a value for this
+	indexedLogsConfirmations = 10
+	batchSize                = 32
```

Comment:
nit. can we be descriptive of what batch size this is (logTriggerRefreshbatchSize)